service/src/test/java/bio/terra/externalcreds/dataAccess/WriteTransactionTest.java

```diff
@@ -0,0 +1,101 @@
+package bio.terra.externalcreds.dataAccess;
+
+import bio.terra.externalcreds.BaseTest;
+import java.util.concurrent.CyclicBarrier;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.CannotSerializeTransactionException;
+import org.springframework.dao.RecoverableDataAccessException;
+
+public class WriteTransactionTest extends BaseTest {
```

Comment:
Since this doesn't need the mvc, should we just put `@SpringBootTest` and `@ActiveProfiles("human-readable-logging")`? I'll clean up the base test situation after this lands.